test/e2e/Makefile

```diff
@@ -1,4 +1,3 @@
-COMETBFT_BUILD_OPTIONS += badgerdb,rocksdb,clock_skew,bls12381
```

Comment:
This means the bls is disabled from the e2e by default? Will they run when they are enabled?

Reply:
Reverted. good catch 👍

Reply:
actually CI is failing because of this line